en/docs/administration/configure-custom-domain.md

```diff
@@ -0,0 +1,27 @@
+# Configure Custom Domain
+
+A custom domain is essential for proper branding, discoverability, and credibility of your site. Choreo allows you to configure a custom domain for your organization's developer portal.
+
+Follow the steps below to configure the custom domain for the developer portal:
+
+## Prerequisites
+
+1. Sign in to the Choreo Console at <a>https://console.choreo.dev/</a> using a Google/ GitHub/ Microsoft account.
+2. If you are a new user, create an organization. Enter a unique organization name and create an organization. For example, "Stark Industries".
+    
+    ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
+
+3. Obtain a custom domain through a domain registrar. 
+4. Through your DNS administrator, create a `CNAME` type DNS record and associate it with the Choreo custom developer access domain: `customdns.devportal.choreo.dev`. 
+5. Optionally, create TLS certificates and keys to enable TLS for the custom domain. For testing purposes, you can also opt to use the `let's encrypt` option. This option allows Choreo to create and manage the certificates on your behalf.
+
+## Configure a custom domain for your Choreo Developer portal
+
+1. Click **Settings** on the left pane. 
```

Comment:
When finalizing terminology, we decided to go with "left navigation menu" for the left pane [1], shall we use that here?
[1] https://docs.google.com/document/d/1C6t7fRbr2rXLzvdrv1wZLJkfqFY8MWR1SAzsUWoNtj0/edit#heading=h.od8e02m15obb

Reply:
done